fhir2dataset/query.py

```diff
@@ -289,22 +277,22 @@ def _join_2_df(
             how = "inner"
 
         if alias_1 == alias_parent:
-            ### to delete after ??
+            ### to delete after ??  # noqa
```

Comment:
why the three # ?

Reply:
Only for the same reason as above

Reply:
there is a special syntax your code editor understands, the famous #TODO! it should turn blue or something is vscode/pycharm. eg:
```
#TODO delete me
```

Reply:
I really like the idea! It's changed.